resources/scss/ceres/widgets/Checkout/_payment-and-shipping-list-widget.scss

```diff
@@ -0,0 +1,43 @@
+@mixin widget-list-box-shadow($color)
```

Comment:
@cjohannsen Bitte in eigene Datei unter /resources/scss/widgets/mixins/ auslagern